README.md

```diff
@@ -237,469 +103,115 @@ You need to copy the following files:
+Game scripts can be added via the 'Check Online Content' button in the main menu.
+
+See also the wiki [guide to game scripts](https://wiki.openttd.org/Game_script) and [TT-Forums graphics game script section](https://www.tt-forums.net/viewforum.php?f=65).
+
+### 1.6) OpenTTD directories
+
+OpenTTD uses its own directory structure to store game data, add-on content etc.
+
+For more information, see the [directory structure guide](./docs/directory_structure.md)
+
+### 1.7) Compiling OpenTTD
+
+If you want to compile OpenTTD from source, instructions can be found in [COMPILING.md](./COMPILING.md)
+
+
+## 2.0) Contact and Community
+
+'Official' channels
+
+- [OpenTTD website](https://www.openttd.org)
+- IRC chat using #openttd on irc.oftc.net [more info about our irc channel](https://wiki.openttd.org/Irc)
+- [OpenTTD on Github](https://github.com/openTTD/) for code repositories and for reporting issues
+- [forum.openttd.org](forum.openttd.org) - the primary community forum site for discussing OpenTTD and related games
+- [OpenTTD wiki](https://wiki.openttd.org/) community-maintained wiki, including topics like gameplay guide, detailed explanation of some game mechanics, how to use add-on content (mods) and much more
+
+'Unofficial' channels
+
+- the OpenTTD wiki has a [page listing OpenTTD communities](https://wiki.openttd.org/Community) including some in languages other than English
+
+
+### 2.1) Contributing to OpenTTD
+
+We welcome contributors to OpenTTD.  More information for contributors can be found in [CONTRIBUTING.md](./CONTRIBUTING.md)
+
+
+### 2.2) Reporting bugs
+
+Good bug reports are very helpful.  We have a [guide to reporting bugs](./CONTRIBUTING.md#bug-reports) to help with this.
+
+Desyncs in multiplayer are complex to debug and report (some software development skils are required).
+Instructions can be found in [debugging and reporting desyncs](./docs/debugging_desyncs.md).
+
+
+### 2.3) Translating
+
+OpenTTD is translated into many languages.  Translations are added and updated via the [online translation tool](https://translator.openttd.org).
+
+
+## 3.0) Licensing
+
+OpenTTD is licensed under the GNU General Public License version 2.0.
+For the complete license text, see the file '[COPYING.md](./COPYING.md)'.
+This license applies to all files in this distribution, except as noted below.
+
+The squirrel implementation in src/3rdparty/squirrel is licensed under the Zlib license.
+See src/3rdparty/squirrel/COPYRIGHT for the complete license text.
+
+The md5 implementation in src/3rdparty/md5 is licensed under the Zlib license.
+See the comments in the source files in src/3rdparty/md5 for the complete license text.
+
+The implementations of Posix getaddrinfo and getnameinfo for OS/2 in src/3rdparty/os2 are distributed partly under the GNU Lesser General Public License 2.1, and partly under the (3-clause) BSD license.
+The exact licensing terms can be found in src/3rdparty/os2/getaddrinfo.c resp. src/3rdparty/os2/getnameinfo.c.
```

Comment:
I think some of these could do with some \`` wrapping them. definitely `getaddrinfo` & `getnameinfo`. Maybe all the paths as well?